cycledash/genotypes.py

```diff
@@ -108,9 +110,15 @@ def get(vcf_id, query, with_stats=True, truth_vcf_id=None):
         q = _add_filters(q, g, query.get('filters'))
         q = _add_orderings(q, g, query.get('sortBy'))
         q = _add_paging(q, g, query.get('limit'), query.get('page'))
+<<<<<<< Updated upstream
         q = q.order_by(asc(func.length(g.c.contig)),
                        asc(g.c.contig),
                        asc(g.c.position))
+=======
```

Comment:
did anything actually change here? `git diff` might just be getting confused by the new blank line above.